Common/Deployment/RemoteMonitoring.json

```diff
@@ -122,7 +122,7 @@
                 "promotionCode": null
             },
             "properties": { }
-        },
+            },
```

Comment:
This looks like merge issue with spacing of braces.  Can you double check these are correct?